roles/srv-announcer/templates/srv-announcer.service.j2

```diff
@@ -0,0 +1,32 @@
+[Unit]
+Description={{ service_name }}.{{ item.key }} - Publish an SRV record on the DNS server
+
+
+After=network-online.target
+Wants=network-online.target
+
+
+[Service]
+TimeoutStartSec=4
+Restart=always
+
+DynamicUser=yes
+
+Environment="AWS_ACCESS_KEY_ID={{ announcer_aws_key_id }}" \
+            "AWS_SECRET_ACCESS_KEY={{ announcer_aws_access_key }}" \
+            "AWS_DEFAULT_REGION={{ announcer_aws_region }}"
```

Comment:
We should never put secrets inside unit files. Use `EnvironmentFile`, and place secrets in the file referenced there.

Reply:
Also, https://github.com/systemd/systemd/pull/16568 might be relevant once merged (and it doesn't hurt to be aware of it before)

Reply:
[Done](https://github.com/wireapp/ansible-sft/pull/8/commits/e38b884e14a237ab0fb11d67670196026dc044f4).